.github/ISSUE_TEMPLATE/dev_client_installation.yml

```diff
@@ -0,0 +1,56 @@
+name: "\U0001F4F1 Development Client Installation/Setup"
+description: 'Report an issue with setting up or creating a Custom Development Client'
+labels: 'needs review', 'Custom Client - Installation'
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to file a bug report! Please fill out this form as completely as possible.
+  - type: markdown
+    attributes:
+      value: If you leave out sections there is a high likelihood your issue will be closed. If you have a question, not a bug report, please post it on our [forums](https://forums.expo.io/) instead.
+  - type: textarea
+    attributes:
+      label: Summary
+      description: Describe the issue in 1 or 2 sentences
+      placeholder: Clearly describe what the expected behavior is vs. what is actually happening when trying to set up your Development Client. This should be as short as possible, while still communicating all the necessary information. If your summary is just 'X library/method isn't working', then you need to continue debugging yourself and provide more information.
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!
```

Comment:
The run commands create these directories so that isn't a sufficient test.  It's more if you make changes to those directories directly rather than via app.json or config plugins